### DISCUSSION TITLE
在2.4版本中，哪里有inverted index的例子

### DISCUSSION THREAD

**Discussion by douxiaofeng99** [milvus-io/milvus#33482] [Q&A and General discussion]
在文档中，发现支持inverted index，我理解这类似于ES的倒排索引。但翻了好多地方，没有找到例子。我想问，对于使用这种索引，在文档如库时，是milvus进行这个字段的自动分词，还是使用者进行分词，然后在检索时，是否可以指定分词器？
或者给一个完整的例子。谢谢
另外，对于此索引，我们测试发现有大小写区分，ES对于这个进行了处理，希望Milvus也进行处理

**Comment by xiaofan-luan** [comment]
After introduce tantivy, we do have a lucene like core to support inverted index. 
But now it is ready to use and still under developement.  So there is no way to specify tokenizer yet.
Wait for next major release

**Reply by zhu3359** [reply]
@xiaofan-luan 现在2.5可以支持倒排了吗？有没有文档或者demo可以参考下

**Reply by yhmo** [reply]
@zhu3359 
https://github.com/milvus-io/pymilvus/blob/1b555d3b9dbb87e18464f9bf1863e660e1cd4a06/examples/orm/inverted_index_example.py#L46

**Reply by zhu3359** [reply]
@yhmo 这里说的不是针对某个标量字段建立倒排的功能，是如 @douxiaofeng99 提出的，针对某个字段自动分词，按照分词term建立倒排索引的功能。

**Reply by xiaofan-luan** [reply]
https://milvus.io/docs/keyword-match.md
